### ISSUE TITLE
Darkmoon Faire Portal

### ISSUE THREAD

**Issue by GeneralCaveman** [HellscreamWoW/Tracker#949]
http://wowwiki.wikia.com/Darkmoon_Faire

According to this, the Darkmoon Faire portal should appear for this week but currently doesn't.

**Comment by Malae1** [comment]
I have seen many players in the Darkmoon Faire. They have reached it through Mulgore's entrance, however there should also still be one for Alliance located in Goldshire when the week of the faire is here.

**Comment by GeneralCaveman** [comment]
So I presume most of the players you have seen would be Horde then? How well scripted is the faire currently as I haven't seen any bug reports on it.

**Comment by Malae1** [comment]
That is correct, a majority of players are horde. I cannot say for certain how many working/bugged quests there, I would say anything that has a special requirement other than looting/killing most likely is having issues.

**Comment by GeneralCaveman** [comment]
As a suggestion, when the team gets around the Darkmoon Faire, is it possible to leave it open for awhile?

Having a limited time to test and having to wait a month between bug reports/being able to test again isn't all that efficient.

**Comment by Malae1** [comment]
I believe this is possible, It may already be this way, but uncertain on that.
